pkg/router/template/plugin.go

```diff
@@ -314,7 +328,7 @@ func createRouterEndpoints(endpoints *kapi.Endpoints, excludeUDP bool, lookupSvc
 						ep.ID = fmt.Sprintf("ept:%s:%s:%d", endpoints.Name, a.IP, p.Port)
 					}
 				} else {
-					ep.TargetName = ep.IP
```

Comment:
I'm assuming `ep.IP` and `a.IP` are the same for when endpoints are being used, but are not the same when endpoint slices are being used?

Reply:
> I'm assuming `ep.IP` and `a.IP` are the same for when endpoints are being used, but are not the same when endpoint slices are being used?

This looks out of date. (I have been pushing a lot recently so we should review again.)